### DISCUSSION TITLE
PLease can you create a Video Tutorial for the Configuring the Instagram RSS Feed

### DISCUSSION THREAD

**Discussion by hussainspams** [DIYgod/RSSHub#10856] [Q&A]
### What feature is it?

PLease can you just create a video tutorial for configuring Instagram to get RSS feeds

### What problem does this feature solve?

I am a accountant by profession. I read all the documentation material but I am just not able to understand. It is going over my head. This will help to simplify the process of adding the instagram user to RSS Hub to get RSS Feeds

### This is not a duplicated feature request or new RSS proposal

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this feature has not already been requested and this is not a [new RSS proposal](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_en.yml).

**Comment by TonyRL** [comment]
Simply replace the username of an instragram user following the example in https://docs.rsshub.app/en/usage.html

**Reply by hussainspams** [reply]
Sorry Let me further elaborate:
I want feeds for user on instagram A. But on self host instance I we need to configure my user i.e. Hussain's user. Now which file I need to update to enter my i.e. Hussain's User ID and password to not get the login error.

**Reply by TonyRL** [reply]
It depends on how you set up your self-hosted instance.
Please take a look at Configuration in https://docs.rsshub.app/en/install which match your setup.

**Reply by hussainspams** [reply]
I have done the setup using the method "Manual Configuration". More specifically using the "yarn start" command. Please can you tell me the path of the file and the name of the file which I have to update my i.e. Hussain's intagram user ID and password.

**Reply by TonyRL** [reply]
> Create a `.env` file in the root directory of your project. Add environment-specific variables on new lines in the form of `NAME=VALUE`. For example:
>
> ```
> CACHE_TYPE=redis
> CACHE_EXPIRE=600
> ```
https://docs.rsshub.app/en/install/#manual-deployment-configuration
https://docs.rsshub.app/en/install/#configuration-route-specific-configurations

**Comment by hussainspams** [comment]
Hey TonyRL,
Please correct me where I am going wrong:

I am at the root directory of my project i.e. /home/ubuntu/RSSHub   
from this location I have created a file env.env by giving the command "touch env.env"
once the file is created I have edited the file by giving the command "nano env.env"
This have opened the Editor & I have entered as follows:
IG_USERNAME=mageni2955
IG_PASSWORD=*************
Then saved the file and closed it.

Please can you check where I am going wrong coz it is still giving me the login error.
Thank you for your patience and time in explaining me like a teaching to a 5 year old.

**Reply by TonyRL** [reply]
> Create a `.env` file ...

not `env.env`

**Comment by hussainspams** [comment]
When I try to create a file .env by giving the command "touch .env" - No file is generated when I give the command "ls -l" subsequently I am not able to user the nano command to edit it. Please can you guide me how to create ".env" file from the command line of terminal.

**Reply by TonyRL** [reply]
`ls -l` won't list out files starting with `.`

**Reply by hussainspams** [reply]
okay I get you. I did the command "ls -l -a" then I was able to see the .env file which I had created.
After that I gave the command "nano .env" and updated the values as below:
IG_USERNAME=mageni2955
IG_PASSWORD=*************

AND FINALLY!!!!!! IT is DONE!!!!
I was able to get the feeds from instagram

**Comment by hussainspams** [comment]
TonyRL Thank you for your support. As an accountant I feel soooo much satisfied that I was able to get it done. (relalising as I am typing that it is supposed to be a peice of cake for the people in the delelopment community.)
Now my next steps are as follow:
1. I will delete the instance that I had created.
2. Then using the video recorder I will again create the instance.
3. Share with you the link of the video to help the people in the community.
Again TonyRL thank you for explaining to me very small small stuff.

**Comment by hussainspams** [comment]
I got this error 
![image](https://user-images.githubusercontent.com/93840468/192461522-ad1c3ef6-9517-4296-8b59-3f00c7323143.png)
